lib/generators/frontier_crud_views/templates/index.html.haml

```diff
@@ -17,10 +17,14 @@
     - <%= model_configuration.ivar_collection %>.each do |<%= model_configuration.model_name %>|
       %tr
 <% model_configuration.attributes.select(&:show_on_index?).each do |attribute| -%>
+<% if attribute.type == 'text' -%>
+        %td= truncate(<%= "#{model_configuration.model_name}.#{attribute.name}" %>, length: 30)
+<% else -%>
         %td= <%= "#{model_configuration.model_name}.#{attribute.name}" %>
```

Comment:
Move this out into the model configuration or somewhere so we can test it.

`model_configuration.as_index_string` or something.